e2e/volume_ops_test.go

```diff
@@ -76,7 +76,7 @@ func TestVolumeOptions(t *testing.T) {
 	volname := "testvol"
 	createReq := api.VolCreateReq{
 		Name:   volname,
-		Bricks: []string{gds[0].PeerAddress + ":" + brickPath},
+		Bricks: []string{gds[0].PeerID() + ":" + brickPath},
```

Comment:
Even PeerAddress should work here, isn't it?

Reply:
No. This was changed to supporting only UUID in PR https://github.com/gluster/glusterd2/pull/357
The CLI supports both UUID and peer address.